v1/middleware/auth.go

```diff
@@ -124,6 +124,32 @@ func isBanned(ctx *gin.Context) (bool, error) {
 	return true, nil
 }
 
+func isSuspended(feature string) gin.HandlerFunc {
+	return func(ctx *gin.Context) {
+		client, _ := jwt.ExtractClient(ctx.GetHeader("Authorization"))
+		username := client.MemberUsername
+
+		isSuspended, err := cache.IsCacheExists(username + ":" + feature)
+		if err != nil {
+			logger.Log("failed on getting suspend data from redis: " + err.Error())
+		}
+		if isSuspended {
+			ttl, err := cache.TTL(username + ":" + feature)
+			if err != nil {
+				logger.Log("failed on getting ttl from redis: " + err.Error())
+			} else {
+				rest.ResponseData(ctx, http.StatusLocked, map[string]interface{}{
+					"until": time.Now().Add(time.Second * time.Duration(ttl)).Format("2006-01-02T15:04:05.999Z"),
```

Comment:
```suggestion
					"until": time.Now().Add(time.Second * time.Duration(ttl)).Format(time.RFC3339),
```
seingatku di golang ada package time yg balikin format RFC3339 yg dipake di code ini.
jadi mungkin boleh diubah pake `const` RFC3339 saja daripada menggunakan string date.
atau mungkin ada alasan lain? btw, kalo diubah, minta tolong dicek juga ya jalan atau ngga codenya.

![image](https://user-images.githubusercontent.com/33178140/125230447-fb456180-e302-11eb-9a5a-ac2489c31e44.png)
https://golang.org/pkg/time/

Reply:
kalo menurut @heggies , @agusID . @christyyy-a , @agusID gimana?
harusnya `RFC3339 = "2006-01-02T15:04:05Z07:00" = "2006-01-02T15:04:05.999Z"` kan?

![image](https://user-images.githubusercontent.com/33178140/125233791-86c1f100-e309-11eb-80ab-b78bd3d0d401.png)
https://stackoverflow.com/questions/32592458/why-cant-go-parse-the-time-represented-by-the-provided-formats